### PR TITLE
basic slider component implemented

### LINKED ISSUE #29: create slider component & remove br tags

### PR DESCRIPTION
**Pull request by aishwarya-bm**
1.created basic slider component
2.removed br & replaced with hr tag
3.minor CSS changes

### CODE REVIEW

File: docs/components/input.html [hunk 1]
```diff
@@ -118,7 +118,7 @@ <h4>Input field with helper text</h4>
                 <p>This is basic field with helper text in the form of suggestions to user.</p>
                 <div class="c-example">
                     <div class="input-group helper">
-                        <label>Username </label><br>
+                        <label>Username </label>
```
Comment (shashidwivedi) [inline docs/components/input.html:121]: You can use 'for' and 'id' attributes for a pair of label and input elements, if they are not nested.

**Review by shashidwivedi — APPROVED** [review]
Rest looks good.

File: docs/components/input.html (hunk 1 above)
Comment (aishwarya-bm) [reply docs/components/input.html:121]: Since they might have different values as per need I have not included

**Review by nishasen — APPROVED** [review]
The code looks clean, I saw you removed the <br> tags, rest lookg good.

**Pull request merged**